pouchdb-core/pouchdb-core.d.ts

```diff
@@ -64,12 +75,62 @@ declare namespace PouchDB {
                 ids: RevisionId[];
                 start: number;
             }
+
+            /** Attachments where index is attachmentId */
+            _attachments?: Attachments;
+        }
+
+        interface AttachmentResponse {
+            content_type: string;
+
+            /** Hash ? */
+            digest: string;
+
+            /** Only present if `attachments` was `false`. */
+            stub?: boolean;
+
+            /** Only present if `attachments` was `false`. */
+            length?: number;
+
+            /** 
+             * Only present if `attachments` was `true`.
+             * {string} if `binary` was `false`
+             * {Blob|Buffer} if `binary` was `true`
+             */
+            data?: string | Blob | Buffer;
```

Comment:
I'd have this use `Attachment` and update that definition (it's used elsewhere, as in `putAttachment`).